config/qa.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  SEGMENT_IO_API_KEY: 'QBtLgV8vCiuRX1lDikbMjcoe9aCHkF6n',
+  SERVER_API_KEY: '79b2d5eb-c1fd-42c4-9391-6b2c9780d591',
```

Comment:
@jmgasper Expose these KEY's is expected?

Reply:
@luizrrodrigues - Probably not.  @rakibansary - Can you take a look please?  I think you added these.

Reply:
@luizrrodrigues @jmgasper, we've always had these

![image](https://github.com/topcoder-platform/community-app/assets/1445965/1abc6d5a-7124-4c52-b10d-7170d456e2b3)

But I agree, we could improve our implementation to reduce the risk of exposing such sensitive information. But it might not be a quick lift.

Reply:
Okay, let follow up with that for now because this already present in dev config.

Can we raise a ticket in Jira to track this?